torch/nn/modules/activation.py

```diff
@@ -963,16 +963,20 @@ def forward(self, query: Tensor, key: Tensor, value: Tensor, key_padding_mask: O
     Args:
         query: Query embeddings of shape :math:`(L, N, E_q)` when ``batch_first=False`` or :math:`(N, L, E_q)`
             when ``batch_first=True``, where :math:`L` is the target sequence length, :math:`N` is the batch size,
-            and :math:`E_q` is the query embedding dimension ``embed_dim``. Queries are compared against
-            key-value pairs to produce the output. See "Attention Is All You Need" for more details.
+            and :math:`E_q` is the query embedding dimension ``embed_dim``. Queries can also be unbatched with shape
```

Comment:
Wording suggestion:
```
query: Query embeddings of shape :math:`(L, E_q)`, :math:`(L, N, E_q)` with ``batch_first=False`` or :math:`(N, L, E_q)`
       with ``batch_first=True``, where :math:`L` is the target sequence length, :math:`N` is the batch size,
       and :math:`E_q` is the query embedding dimension ``embed_dim``
```

wdyt about the idea of just including the unbatched option within the list of allowed shapes?

Reply:
Slightly tweaked that to 
```
:math:`(L, E_q)` for unbatched input, :math:`(L, N, E_q)` with ``batch_first=False`
```

because without that 
```
Query embeddings of shape :math:`(L, E_q)`, :math:`(L, N, E_q)` with ``batch_first=False``
```

seems like :math:`(L, E_q)` should be passed with `batch_first=False`.